content/en/blog/_posts/2024-03-15-userns-beta/index.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: blog
+title: "Kubernetes 1.30: User-namespace hits Beta!"
+date: 2024-03-13
```

Comment:
I used this date as a placeholder. If there is any date I should use, let me know